scripts/gen_flatbuffer.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+echo "Running gen_flatbuffers.sh OSTYPE is $OSTYPE"
+ROOT=$(pwd)
+FF_LOCATION="$ROOT/third_party/flatbuffers"
+cd "$FF_LOCATION" || exit
+mkdir build
+cd build || exit
+py() { command python "$@"; }
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_OSX_ARCHITECTURES=x86_64 ..
+cmake --build . --target flatc
+if [[ "$OSTYPE" == "darwin"* ]]; then
+   # for some reason on mac the binary appears in a different directory...
+   FLATC=./Debug/flatc
+else
```

Comment:
If `CMAKE_GENERATOR` is reused, it should not be necessary (or this condition should be `if [[ $CMAKE_GENERATOR == Xcode ]];`